apps/web_delivery/web/views/error_view.ex

```diff
@@ -0,0 +1,9 @@
+defmodule WebDelivery.ErrorView do
+  use WebDelivery.Web, :view
+
+  def render("404.json", _assigns), do: "Not found"
+
+  def render("500.json", _assigns), do: "Internal server error"
```

Comment:
Since the request is expecting json, are we allowed to just respond with just a string? Should this be a map instead?